egograph/backend/tests/conftest.py

```
@@ -142,23 +142,34 @@ def duckdb_with_sample_data(duckdb_conn, tmp_path):
     """YouTube用DuckDB接続のラッパー（テスト用の属性を保持）。"""
 
-    def __init__(self, conn, watches_parquet_path, videos_parquet_path):
+    def __init__(self, conn, watch_events_parquet_path, videos_parquet_path):
         self._conn = conn
-        self.test_watches_parquet_path = watches_parquet_path
+        self.test_watch_events_parquet_path = watch_events_parquet_path
         self.test_videos_parquet_path = videos_parquet_path
 
+    @property
+    def test_watches_parquet_path(self) -> str:
+        """後方互換プロパティ（test_watch_events_parquet_path へのエイリアス）。"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)